tools/perfscale-load-generator/README.md

```diff
@@ -0,0 +1,65 @@
+# Perscale Load Generator
+
+The load generator is a tool aimed at stressing kubernetes and kubevirt control plane by creating several VMIs in parallel. The main functionallity it provides can be summarized as follows:
+- Create/delete the VMIs declared in a scenario.
+- Create VMIs in one namespaces, or one VMI per namespace.
+- Wait VMIs be created in each iteration.
+- Wait VMIs be deleted in each iteration.
+
+This tool introduces load into the system and the relevant metrics and results can be collected using the [perfscale-audit tool](../perfscale-audit/).
+
+## CLI
+```
```

Comment:
I wouldn't put the --help output into the readme. It's likely to change at some point and somebody will forget updating it here.
Maybe put that the Params can be found by calling --help?

Reply:
As discussed yesterday in the Sig-scale meeting, I will move the options to a config file